doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -25,15 +25,131 @@
 Traceability to Reference Model.
 
 <a name="2.3"></a>
-## 2.3 Requirements.
+## 2.3 Containerised Platform Requirements.
+
+The Reference Model (RM) defines the NFVI to consist of the physical resources, virtualised resources and a software managment system.  In the virtualised world, the NFVI Software Management System (Virtualised Infrastructure Manager -- VIM) consists of the Guest Operating System, Hypervisor and, if needed, other software such as libvirt.  And the VIM is responsible for, among others, tenant management, resources management, inventory, scheduling, and access management. Now consider, the containerisation equivalent.
```

Comment:
Guest operating system, hypervisor and libvirt are part of NFVI, not VIM. VIM is the piece which manages the NFVI.

Reply:
@CsatariGergely Correct. As per RM chapter "the NFVI Software Management System consists of Virtualised Infrastructure Manager (VIM), the Guest Operating System, Hypervisor and, if needed, other software such as libvirt."

Reply:
Just reading this again - I'm not sure we got agreement?  This block seems to suggest VIM (NFVI Software Managemenet System) is part of NFVI

> NFVI to consist of the physical resources, virtualised resources and a software managment system

But VIM is separate to NFVI.  Could the following be suitable?

> The Reference Model (RM) defines the NFVI to consist of NFVI Hardware (e.g. the physical resources) and NFVI Software (e.g. the Guest Operating System, Hypervisor and, if needed, other software such as libvirt). The NFVI is managed by a Virtual Infrastructure Manager (VIM), which is responsible for, among others, tenant management, resources management, inventory, scheduling, and access management. Now consider, the containerisation equivalent.

Reply:
FYI I've raised #559 to address this in the RM.